File: apps/server-car-rental/src/APIs/Car/Dtos/CarCreateInput.cs
namespace CarRentalService.APIs.Dtos;

public class CarCreateInput
{
    public DateTime CreatedAt { get; set; }
    public ICollection<RentalDto>? Rentals { get; set; }
    public decimal? RentalPricePerDay { get; set; }
    public string? RegistrationNumber { get; set; }
    public string? Make { get; set; }
    public string? Model { get; set; }
}

File: apps/admin-car-rental/src/api/car/CarCreateInput.ts
import { RentalCreateNestedManyWithoutCarsInput } from "./RentalCreateNestedManyWithoutCarsInput";

export type CarCreateInput = {
  rentals?: RentalCreateNestedManyWithoutCarsInput;
  rentalPricePerDay?: number | null;
  status?: "Option1" | null;
  registrationNumber?: string | null;
  make?: string | null;
  model?: string | null;
};
